src/ui/report-dialog.html

```diff
@@ -0,0 +1,115 @@
+<!--
+ *  Copyright (c) 2015 The WebRTC project authors. All Rights Reserved.
+ *
+ *  Use of this source code is governed by a BSD-style license
+ *  that can be found in the LICENSE file in the root of the source
+ *  tree.
+-->
+<link rel="import" href="../../polymer/polymer.html">
+<link rel="import" href="../../paper-dialog/paper-dialog.html">
+<link rel="import" href="../../paper-button/paper-button.html">
+
+<polymer-element name="report-dialog">
+  <template>
+    <!-- Report dialog -->
+    <paper-dialog id="mainDialog" heading="File a bug" autoCloseDisabled backdrop transition="paper-dialog-transition-center"
+    closeSelector="[dismissive]">
+      <paper-input-decorator floatingLabel label="Describe your issue here">
+        <paper-autogrow-textarea id="textAreaAutoGrow">
+          <textarea id="textArea" value="{{description}}"></textarea>
+        </paper-autogrow-textarea>
+      </paper-input-decorator>
+      <h3>Disclaimer</h3>
+      <p>The report will contain information about your device including network information that is useful to troubleshoot the issue and will be available for a period of 90 days.</p>
+      <paper-button affirmative on-tap="{{upload}}" disabled="{{submitting}}">Upload report</paper-button>
+      <paper-button dismissive disabled="{{submitting}}">Cancel</paper-button>
+    </paper-dialog>
+
+    <!-- Report upload success -->
+    <paper-dialog id="success" heading="Success" autoCloseDisabled transition="paper-dialog-transition-center" closeSelector="[affirmative]">
+      <p><a href="{{reportUrl}}">This is the link to your report</a></p>
```

Comment:
Normal users may not know how to easily copy the link from here... but lets handle that after this is in.